VAMobile/src/screens/BenefitsScreen/Letters/BenefitSummaryServiceVerification/BenefitSummaryServiceVerification.test.tsx

```diff
@@ -93,128 +86,34 @@ context('BenefitSummaryServiceVerification', () => {
-    })
-  })
-
-  describe('on include military service info switch click', () => {
-    it('should update that switches on value', async () => {
-      const switchIcon = testInstance.findAllByType(Switch)[0]
-      const rnSwitch = testInstance.findAllByType(RNSwitch)[0]
-
-      switchIcon.props.onPress()
-      expect(rnSwitch.props.value).toEqual(false)
-
-      switchIcon.props.onPress()
-      expect(rnSwitch.props.value).toEqual(true)
-    })
-  })
-
-  describe('on monthly award switch click', () => {
-    it('should update that switches on value', async () => {
-      const switchIcon = testInstance.findAllByType(Switch)[1]
-      const rnSwitch = testInstance.findAllByType(RNSwitch)[1]
-
-      switchIcon.props.onPress()
-      expect(rnSwitch.props.value).toEqual(false)
-
-      switchIcon.props.onPress()
-      expect(rnSwitch.props.value).toEqual(true)
-    })
-  })
-
-  describe('on combined service connected rating switch click', () => {
-    it('should update that switches on value', async () => {
-      const switchIcon = testInstance.findAllByType(Switch)[2]
-      const rnSwitch = testInstance.findAllByType(RNSwitch)[2]
-
-      switchIcon.props.onPress()
-      expect(rnSwitch.props.value).toEqual(false)
-
-      switchIcon.props.onPress()
-      expect(rnSwitch.props.value).toEqual(true)
-    })
-  })
-
-  describe('on permanently disabled switch click', () => {
-    it('should update that switches on value', async () => {
-      const switchIcon = testInstance.findAllByType(Switch)[3]
-      const rnSwitch = testInstance.findAllByType(RNSwitch)[3]
-
-      switchIcon.props.onPress()
-      expect(rnSwitch.props.value).toEqual(false)
-
-      switchIcon.props.onPress()
-      expect(rnSwitch.props.value).toEqual(true)
-    })
-  })
-
-  describe('on one or more disability switch click', () => {
-    it('should update that switches on value', async () => {
-      const switchIcon = testInstance.findAllByType(Switch)[4]
-      const rnSwitch = testInstance.findAllByType(RNSwitch)[4]
-
-      switchIcon.props.onPress()
-      expect(rnSwitch.props.value).toEqual(false)
-
-      switchIcon.props.onPress()
-      expect(rnSwitch.props.value).toEqual(true)
-    })
+    expect(screen.getByText('Army')).toBeTruthy()
+    expect(screen.getByText('Honorable')).toBeTruthy()
+    expect(screen.getByText('January 01, 1990')).toBeTruthy()
+    expect(screen.getByText('October 01, 1993')).toBeTruthy()
+    expect(screen.getByText('Your current monthly payment is $123.00. The effective date of the last change to your current payment was June 06, 2013.')).toBeTruthy()
+    expect(screen.getByText('Your combined service-connected rating is 88%.')).toBeTruthy()
+    expect(screen.getByText("You aren't considered to be totally and permanently disabled solely due to your service-connected disabilities.")).toBeTruthy()
+    expect(screen.getByText("You don't have one or more service-connected disabilities.")).toBeTruthy()
   })
```

Comment:
I haven't looked too deep into this, but aren't some of these tests still needed? Some of this functionality still remains for the screen

Reply:
No I don't think they should. It isn't testing a ui change aside from the switch value changing. That should be covered in the Switch unit tests and this would just be duplicate unit tests that should be covered by the component. If it was covering ui changes then I could see keeping them.